ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -6,6 +6,7 @@ package power
 import (
 	"context"
 	"encoding/base64"
+	b64 "encoding/base64"
```

Comment:
Why duplicate imports with alias?

Reply:
ok that's a typo, I'll remove the duplicate